Sources/VLCPlaybackController.m

```diff
@@ -834,6 +833,8 @@ - (void)switchAspectRatio
     if ([self.delegate respondsToSelector:@selector(showStatusMessage:)]) {
         [self.delegate showStatusMessage:[NSString stringWithFormat:NSLocalizedString(@"AR_CHANGED", nil), [self stringForAspectRatio:_currentAspectRatio]]];
     }
+
+    [_delegate playbackControllerDidSwitchAspectRatio:_currentAspectRatio];
```

Comment:
you need to make a check here if the delegate responds to the selector, that call is under an @optional which means that a delegate doesn't need to implement it. Even though you might have a delegate this could crash when the function is not implemented

Reply:
Okay, ahh thats what that does